src/main/java/org/jenkinsci/plugins/saml/SamlPluginConfig.java

```diff
@@ -36,13 +36,14 @@
     private final String usernameCaseConversion;
     private final String usernameAttributeName;
     private final String logoutUrl;
+    private final String binding;
 
     private SamlEncryptionData encryptionData;
     private SamlAdvancedConfiguration advancedConfiguration;
 
     public SamlPluginConfig(String displayNameAttributeName, String groupsAttributeName,
                             int maximumAuthenticationLifetime, String emailAttributeName, String idpMetadata,
-                            String usernameCaseConversion, String usernameAttributeName, String logoutUrl,
+                            String usernameCaseConversion, String usernameAttributeName, String logoutUrl, String binding,
```

Comment:
Is this change backward compatible on config files?

Reply:
yes, it is tested manually and in the tests